public/electron.js

```diff
@@ -91,6 +91,10 @@ function createWindow () {
             }
           }},
           { type: 'separator' },
+          { label: 'Open dev tools', click: function() {
```

Comment:
Maybe we should add it under a menu called `Advanced dev tools`, so users won't click on it by accident.